charts/sftd/templates/ingress.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
```

Comment:
suggestion: not sure which k8s is now the minimal required/compatible one, but if we go with a more recent one the should probably changed to `networking.k8s.io/v1`

Reply:
that one was introduced only in `1.19` I think. Let me double check

Reply:
It seems to be only introduced in 1.19.     Kubespray currently uses 1.18 as its stable kubernetes version. So we can not use this yet; alas.

https://kubernetes.io/blog/2020/08/26/kubernetes-release-1.19-accentuate-the-paw-sitive/